.circleci/setup-virtualenv.sh

```diff
@@ -26,12 +26,7 @@ shift || :
 
 # Tell pip where it can find any existing wheels.
 export PIP_FIND_LINKS="file://${WHEELHOUSE_PATH}"
-
-# It is tempting to also set PIP_NO_INDEX=1 but (a) that will cause problems
-# between the time dependencies change and the images are re-built and (b) the
-# upcoming-deprecations job wants to install some dependencies from github and
-# it's awkward to get that done any earlier than the tox run.  So, we don't
-# set it.
+export PIP_NO_INDEX="1"
```

Comment:
For (a) we now have a command-line tool that we can run on-demand to re-build the images when we want against whichever branch we want.  For (b) we removed "upcoming-deprecations" from CI a while ago.